person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
 
```

Comment:
- [ ] To properly implement the `has-many` relationship between `Person` and `Rental`, getters and setters for `rentals` should be present